pinot-core/src/main/java/org/apache/pinot/core/query/request/ServerQueryRequest.java

```diff
@@ -94,10 +94,12 @@ public ServerQueryRequest(InstanceRequest instanceRequest, ServerMetrics serverM
     if (aggregationsInfo != null) {
       _aggregationExpressions = new HashSet<>();
       for (AggregationInfo aggregationInfo : aggregationsInfo) {
-        if (!aggregationInfo.getAggregationType().equalsIgnoreCase(AggregationFunctionType.COUNT.getName())) {
-          for (String expressions : AggregationFunctionUtils.getArguments(aggregationInfo)) {
-            _aggregationExpressions.add(TransformExpressionTree.compileToExpressionTree(expressions));
-          }
+        String aggregationType = aggregationInfo.getAggregationType();
```

Comment:
I would recommend specialize THETA_SKETCHES, so that it won't affect DISTINCT

Reply:
Yeah, I was debating between the two approaches. I missed that we could support expressions in distinct. Will update to special case theta-sketch.